spec/models/container_image_spec.rb

```diff
@@ -42,4 +42,25 @@
     expect(image.computer_system).not_to be_nil
     expect(image.operating_system).to eq(image.computer_system.operating_system)
   end
+
+  context "#annotate_deny_execution" do
+    require 'kubeclient'
```

Comment:
Perhaps_ this should be in app/models/container_image.rb and then not needed here.

Reply:
I still think this should be in container_image.rb.  That file mentions `KubeException` so needs it.
require in spec is frequently red flag for missing require in code...

LGTM otherwise.